packages/editor-ui/src/components/banners/__tests__/__snapshots__/V1Banner.spec.ts.snap

```diff
@@ -2,69 +2,209 @@
 
 exports[`V1 Banner > should render banner 1`] = `
 <div>
-  <n8n-callout
-    class="callout v1container"
+  <div
```

Comment:
This snapshot has changed quite drastically. Is that intentional?

Reply:
It's the side effect of using our render helper which has all the components and directives installed (this is what we want to introduce also in design system)